napari/components/viewer_model.py

```diff
@@ -10,16 +10,18 @@
     TYPE_CHECKING,
     Any,
     Dict,
+    Iterator,
     List,
-    Mapping,
     Optional,
     Sequence,
     Set,
     Tuple,
+    Type,
     Union,
 )
 
 import numpy as np
+from app_model.expressions import Context
```

Comment:
Should/could this import (and al the napari.types below) be hidden behind `if TYPE_CHECKING`?

Reply:
As `from napari._app_model.context import create_context` is lazily I guess yes.

Reply:
Added a commit that do so.